File: Exemplos/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/BRIDGE/IniciaOperacaoAdministrativa.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.BRIDGE

import com.elgin.intent_digital_hub.IntentDigitalHubService.BRIDGE.BridgeCommand

class IniciaOperacaoAdministrativa(
    private val idTransacao: Int,
    private val pdv: String,
    private val operacao: Int
) : BridgeCommand("IniciaOperacaoAdministrativa") {
    override fun functionParameters(): String {
        return "\"idTransacao\"" + ":" + idTransacao + "," +
                "\"pdv\"" + ":" + "\"" + pdv + "\"" + "," +
                "\"operacao\"" + ":" + operacao
    }
}

File: Exemplos M10/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/BRIDGE/inicia_operacao_administrativa.dart
import 'package:intent_digital_hub/IntentDigitalHubService/BRIDGE/bridge_command.dart';

class IniciaOperacaoAdministrativa extends BridgeCommand {
  final int _idTransacao;
  final String _pdv;
  final int _operacao;

  IniciaOperacaoAdministrativa(this._idTransacao, this._pdv, this._operacao)
      : super('IniciaOperacaoAdministrativa');

  @override
  functionParameters() {
    return '"idTransacao"'
        ':'
        '$_idTransacao'
        ','
        '"pdv"'
        ':'
        '"$_pdv"'
        ','
        '"operacao"'
        ':'
        '$_operacao';
  }
}
